src/main/ansible/vagrant.yml

```diff
@@ -23,29 +23,31 @@
     easy_test_users_email: "{{ deployer_email }}"
     easy_test_users_ldapadmin_password: "{{ easy_ldap_dir_ldapadmin_password }}"
   roles:
+    - dans.capture-deployer-email
     - dans.local-test-vm-base
+    - dans.local-yum-repo
     - dans.easy-ldap-dir
-    - dans.capture-deployer-email
     - dans.easy-test-users
-    - dans.local-yum-repo
 
 - hosts: "test"
   become: yes
   tasks:
-    - name: Installing package
-      yum:
-        name: dans.knaw.nl-easy-sword2
-        state: latest
+  - name: Create directories
+    file:
+      path: "{{ item }}"
+      recurse: yes
+      state: directory
+      mode: "0777"
+    with_items:
```

Comment:
I am in favor of keeping the creation of these default directories in the RPM post install.